magento/devdocs#: Cloud. ERROR: Unable to send mail: Unknown error. New Topic. (#5356)
magento/devdocs: guides/v2.2/cloud/project/project-webint-basic.md

Configure your email notifications, services, and more as needed through the Magento Admin and test emails.

{: .bs-callout-warning }

> If **Outgoing emails** option is equal to Off then [Unable to send mail: Unknown error]({{ site.baseurl }}/cloud/trouble/unable-send-email.html) will be displayed.

Reviewer: To avoid using "will be" and make it read more like a real warning, consider changing the order...

```suggestion
You may receive an [Unable to send mail: Unknown error]({{ site.baseurl }}/cloud/trouble/unable-send-email.html) message if the **Outgoing emails** option is set to Off.
```

Author: Hi, @hguthrie!

**Done**
